example/timer/odp_timer_accuracy.c

```diff
@@ -291,9 +291,9 @@ static int start_timers(test_global_t *test_global)
 	test_global->timeout_pool = pool;
 
 	if (test_global->opt.clk_src == 0)
-		clk_src = ODP_CLOCK_CPU;
+		clk_src = ODP_CLOCK_DEFAULT;
 	else
-		clk_src = ODP_CLOCK_EXT;
+		clk_src = test_global->opt.clk_src;
```

Comment:
How about replacing this whole if-else with simply "clk_src = test_global->opt.clk_src"? The API promises that ODP_CLOCK_DEFAULT is zero.

Other than the above, this patch set looks good to me.

Reply:
I was thinking that it's more explicit to the user that "-s 0" option is DEFAULT, and for other values it is implementation specific where they connect to. But maybe it's enough that the help text documents it.

Reply:
Fixed in v3.